engine/src/state_chain_observer/sc_observer/mod.rs

```diff
@@ -297,29 +304,13 @@ where
                             for event_record in events {
                                 match_event! {event_record.event, {
                                     state_chain_runtime::RuntimeEvent::Validator(
-                                        pallet_cf_validator::Event::PeerIdRegistered(
-                                            account_id,
-                                            ed25519_pubkey,
-                                            port,
-                                            ip_address,
-                                        ),
-                                    ) => {
-                                        peer_update_sender
-                                            .send(PeerUpdate::Registered(
-                                                    PeerInfo::new(account_id, ed25519_pubkey, ip_address.into(), port)
-                                                )
-                                            )
-                                            .unwrap();
-                                    }
+                                        pallet_cf_validator::Event::PeerIdRegistered(..),
+                                    ) |
                                     state_chain_runtime::RuntimeEvent::Validator(
-                                        pallet_cf_validator::Event::PeerIdUnregistered(
-                                            account_id,
-                                            ed25519_pubkey,
-                                        ),
+                                        pallet_cf_validator::Event::PeerIdUnregistered(..),
```

Comment:
You can delete these branches from the match

Reply:
If you want to log something move the macro into a shared location and use the macro, in the monitor_p2p_registration_events function.

Reply:
The problem is that the macro logs un-matched events with `Ignoring event`.So if used in 2 places, it will log deceptive ignore messages. Was not sure what to do about this.